selfdrive/controls/lib/events.py

```diff
@@ -198,6 +198,14 @@ def calibration_incomplete_alert(CP, sm, metric):
       Priority.LOWER, VisualAlert.none, AudibleAlert.none, 0., 0., 15.),
   },
 
+  EventName.startupWhitePanda: {
+    ET.PERMANENT: Alert(
+      "WARNING: White panda is deprecated",
```

Comment:
The other alert capitalizes panda

Reply:
I see we broke the rule to not capitalize product names in the events. I'll change it back to uppercase for consistency

Reply:
The other startup alerts are not capitalized, so I'll leave this as is to have consistency there.